prow/config/external_plugins_config.yaml

```diff
@@ -210,6 +231,42 @@ ti-community-label:
       - 'do-not-merge/cherry-pick-not-approved'
     exclude_labels:
       - status/can-merge
+  - repos:
+      - pingcap/tidb
+    prefixes:
+      - priority
+      - closed
+      - component
+      - difficulty
+      - epic
+      - feature
+      - severity
+      - sig
+      - status
+      - type
+    additional_labels:
```

Comment:
should we list all labels to here?

Reply:
No, you can refer to https://book.prow.tidb.io/#/plugins/label.

The lables that need to be listed here are those that cannot be classified.